src/main/resources/application.yaml

```diff
@@ -0,0 +1,13 @@
+spring:
+  application:
+    name: springboot_fm_example
+
+rox:
+  key: "<INSERT YOUR SDK KEY HERE>"
+  options:
```

Comment:
As earlier comments - hopefully these options not needed?

Reply:
the app was not running successfully without them. Maybe after Asaf is done with his Java SDK bug fixes we will not need them.